README.md

```diff
@@ -326,10 +345,15 @@ subcommand.
 
 ## Review changes for other operating systems/architectures
 
-The `--system` flag allows setting a system different from the current one. Note
-that the result nix-shell may not be able to execute all hooks correctly since
+The `--systems` flag allows setting a system different from the current one.
+Note that the result nix-shell may not be able to execute all hooks correctly since
 the architecture/operating system mismatches.
 
+By default, `nixpkgs-review` targets only the current system (`--systems current`).
+You can also explicitly provide one or several systems to target (`--systems "x86_64-linux aarch64-darwin"`).
+The `--systems all` value will build for the four major platforms supported by hydra (`--x86_64-linux`, `aarch64-linux`, `x86_64-darwin` and `aarch64-darwin`).
+Ensure that your system is capable of building for the specified architectures, either locally or through the remote builder protocol.
```

Comment:
Could you add please a guide/link to a guide for how you build darwin programs on linux?

Reply:
I think this is it

https://nix.dev/manual/nix/2.18/advanced-topics/distributed-builds

Reply:
To my knowledge, there isn't anything special with darwin compared to other foreign architectures.
You simply need to have remote builders able to build darwin packages (a mac basically).
Then, you can `nix-build` everything you want.

Reply:
Yeah, I mean any foreign architecture. Just darwin is my use case

Reply:
Ok, then the page you linked is all you need.